### DISCUSSION TITLE
Overwrite existing css

### DISCUSSION THREAD

**Discussion by Joseph-Pittera** [ankitskvmdam/clean-jsdoc-theme#261] [Q&A]
Hi !
I'd like to overwrite the list-style: none on ul in the css (to have list-style: disc). I tried to put it on a custom css file, but it doesn't work, the module's css file is given priority.
Or can it be done ?
Thanks for your support

**Comment by ankitskvmdam** [comment]
v5 supports custom CSS via `customCss` (inline) and `customCssFile` (a file), both loaded after the theme stylesheet so they override it. Docs: https://ankdev.me/clean-jsdoc-theme/theme/configuration/ Closing as resolved.
